### DISCUSSION TITLE
Organizing The Starred Section

### DISCUSSION THREAD

**Discussion by Woshimbo** [community/community#4997] [Mobile]
I'm on iOS as a reference.

Would really enjoy if the starred list was treated as bookmarks or a file system. 

Just some simple naming, folders, and being able to nest them would be very beneficial in my opinion.

**Comment by eliperkins** [comment]
Thanks for the feedback @FlippingBytes! We hear ya, this is something the team has been researching a bit. Using stars as a bookmark system is something we've heard from other users as well.

I've jotted your notes down on an internal issue for this. I'll keep you posted with any updates!

**Reply by Woshimbo** [reply]
Thanks for the reply, have a good one!
